netket/models/DS.py

```diff
@@ -0,0 +1,160 @@
+from typing import Union, Tuple, Any, Callable
+
+from netket.utils.types import PyTree
+
+import jax
+from jax import numpy as jnp
+from jax.scipy.special import logsumexp
+from flax import linen as nn
+
+from netket.utils.types import NNInitFunc
+
+from jax.nn.initializers import (
+    zeros,
+    ones,
+    lecun_normal,
+)
+
+
+class DeepSet(nn.Module):
+    k: int
+    """Number of moments for the periodic transformation."""
+    L: jnp.float64
+    sdim: int
+
+    layers_phi: int
+    layers_rho: int
+    """Number of layers in phi/rho network"""
```

Comment:
It's better to not combine docstrings like that, it looks better in the docs to have a separate one for each property.